### PR TITLE
EWL-4104: Create Ribbon

### PR DESCRIPTION
**Pull request by nstriedinger**
**Jira Ticket**

- https://issues.ama-assn.org/browse/EWL-4104

## Description

Create Ribbon

## To Test
- [ ] Verify http://localhost:3000/?p=molecules-menu-user
- [ ] Verifyhttp://localhost:3000/?p=molecules-menu
- [ ] Verify http://localhost:3000/?p=organisms-ribbon
- [ ] Verify mobile and all browsers.

## Visual Regressions

Does your work introduce any visual regressions to existing work in the Style Guide, or in a Drupal 8 environment? Please either call these out here or address them before marking your PR as `ready for review`.

## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

### CODE REVIEW

File: styleguide/source/_patterns/02-molecules/menu.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Description
+This Pattern shows the menu dropdown molecule.
```
Comment (labbydev) [inline styleguide/source/_patterns/02-molecules/menu.md]: This should probably be linking to the variants for the menus, yes?

**Comment by labbydev** [comment]
I can take over some of the updates that need to happen on this PR, but I'd like to get any feedback from @designerneil and @chelsiejohnston before that.

File: styleguide/source/_patterns/01-atoms/unordered-list.json [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-  "unorderedList": [{ 
-    "text": "This is a list item in an unordered list" 
-  },{ 
-    "text": "An unordered list is a list in which the sequence of items is not important. Sometimes, an unordered list is a bulleted list. And this is a long list item in an unordered list that can wrap onto a new line."
-  },{ 
-    "text": "Lists can be nested inside of each other",
-    "sublist" : [{
-      "text":"This is a nested list item"
+  "unorderedList": {
+    "class": "ama__list",
```
[the remark this answers is not in the document]
Comment (designerneil) [reply styleguide/source/_patterns/01-atoms/unordered-list.json]: all text s going to live in json files. Why do you think it should be in the twig file?

**Comment by designerneil** [comment]
@labbydev i'm actively working in this branch. I'm making the changes you requested this morning (12/14). Hopefully we can merge it before EOD

File: styleguide/source/_patterns/01-atoms/unordered-list.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (designerneil) [reply styleguide/source/_patterns/01-atoms/unordered-list.json]: this class makes the unordered list reusable

**Pull request closed without merging**